modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -90,6 +91,12 @@ static struct
 
     /* Icon group file containing one PNG icon and one normal icon */
     {L"%temp%\\ROS.ico", 1, 1, TRUE},
+
+    /* Executable file with bad 'Icon Group' but good 'Icons'.
+     * Windows explorer shows the program's icon correctly in WinXP/Win2K3
+     * but Windows 7 shows only a default icon. This is analogous
```

Comment:
```suggestion
     * but WinVista/Win7 show only a default icon. This is analogous
```

Reply:
I want to leave this and only make this change is other changes are needed. Thanks.